test/release/examples/primers/opaque.chpl

```diff
@@ -7,6 +7,7 @@
    support unstructured arrays like graphs. They are a special case of
    :ref:`associative domains and arrays <primers-associative>`.
 */
+pragma "error mode fatal"
```

Comment:
It seems unfortunate to require the pragma in release/examples tests, and particularly in a primer.  In this specific case, I wonder if the explicit `module` declaration could/should just be removed (most primers don't seem to use them unless they're teaching modules?).  In other cases (benchmarks), I'm wondering whether try! could/should be used instead?

Reply:
Looking at the rendered version of this online makes me think even more that we should just remove the module declaration in this case.  It doesn't seem to have a purpose.

Reply:
I think it's a matter of file naming. We don't want the implicit module to be named `opaque` because that might interfere with things like `domain(opaque);`. If we're willing to rename it to opaque-domains or something like that, I think we can drop the module declaration.

Reply:
In updating it, I'd pull the module declaration anyway I think.  It just clutters the documentation.

Reply:
I have PR #7277 to fix this about the opaque primer.

Reply:
Sorry, I got my wires crossed:  I'd seen your "this doesn't need the pragma" comment on email and thought it was on this comment stream rather than the previous one (which is why my last response probably didn't parse well).  Bummer about the need to dodge the implicit module name, but I'm OK with that PR.

The comment that kicked this stream off was also intended to wonder about what could be done to avoid using the pragma in other release/examples tests as well—I just attached it to this one since it was one of the most user-facing.

Reply:
I can get rid of the test/release/examples/benchmarks/ssca2 ones by removing some module declarations (so we instead use the implicit module). Arguably the module statements make it clearer what is going on, but I think it'd be acceptable to remove them and rely on the implicit default. Sound OK?

Reply:
I think this sounds OK.  If the modified versions in the studies directory is also a performance test, I might do the same there in order to keep it compiling with older compilers.  If not, leaving it as-is is fine.

Reply:
See PR #7287.